celesta-sql/src/main/java/ru/curs/celesta/score/BinaryColumn.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Binary column (IMAGE or BLOB type).
  */
-public final class BinaryColumn extends Column {
+public final class BinaryColumn extends Column<String> {
```

Comment:
I'm slightly in doubt which generic type should be used here ``<String>`` or ``<BLOB>``, therefore I decided not to include column refs into generated cursors for binary type columns.

Reply:
Funny thing is that it really doesn't matter. We need type parameter for Column for type-checked versions of setRange/setFilter, but we cannot set a range or a filter on BLOB field.
So -- really doesn't matter